tests/test_wallet.py

```diff
@@ -606,3 +605,61 @@ def test_hsm_secret_encryption(node_factory, executor):
     os.write(master_fd, password.encode("utf-8"))
     l1.daemon.wait_for_log("Server started with public key")
     assert id == l1.rpc.getinfo()["id"]
+
+
+@unittest.skipIf(VALGRIND, "It does not play well with prompt and key derivation.")
+def test_hsmtools_secret_decryption(node_factory):
+    l1 = node_factory.get_node()
+    password = "reckless\n"
```

Comment:
```suggestion
    password = "reckless\n".encode("utf-8") 
```

?? then you can use it inline elsewhere.

Reply:
Yeah but then I would have to decode it for Pylightning `call()` method, which (since we also need to remove the trailing `\n`) is arguably dirtier ?